app/views/records/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<section>
+  <div class="d-flex align-items-center justify-content-around my-4">
+    <%= link_to "Back", category_records_path(@category), class: "btn btn-secondary" %>
+    <h3>New Transaction</h3>
+    <i></i>
+  </div>
+  <%= form_with model: @record, url: category_records_path, method: :post do |f| %>
+    <div class="mb-3">
+      <%= f.label :name, class: "form-label" %><br />
+      <%= f.text_field :name, class: "form-control" %>
+    </div>
+     <div class="mb-3">
+      <%= f.label :amount, class: "form-label" %><br />
+      <%= f.number_field :amount, class: "form-control" %>
+    </div>
+    <div class="d-flex justify-content-center">
+      <%= f.submit "Create Record", class: "btn btn-primary" %>
+    </div>
+  <% end %>
+</section>
```

Comment:
- [ ] The requirements state that the form page for adding a new _**transaction/record**_ should have a form input where the user can choose at least one existing category that the new record/transaction will belong to. Kindly implement this project requirement.